Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,39 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you swear sometimes?"
+    m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's alright if you swear occasionally, {w=0.2}a lot of people do, after all."
+    m 2rtd "I even heard of some studies stating that using stronger language helps easing the pain if you, say, hurt yourself."
+    m 3eta "It's called the analgesic response, which makes your body more impervous to pain."
+    m 3esc "I know some people have it a natural part of their speech patterns, {w=0.2}and I that your vocabulary is formed by society and those around you."
+    m 3efb "These expressions can make a talk feel a lot more casual, {w=0.2}and even help spice up certain conversations..."
+    m 3rksdlb "But there's also such a thing as too much swearing, I'd say"
```

Comment:
```suggestion
    m 3rksdlb "But there's also such a thing as too much swearing, I would say."
```

Reply:
Any reason why you'd not want to abbreviate "I would"?

Reply:
Not really, just sounds better here in my opinion.

Reply:
It also makes Monika sound like a country grandma in Leeds pouring tea and enunciating every word :P Oh well, not a big deal.

Reply:
No, thanks, I don't think it actually fits here, Monika is the one to abbreviate rather than keep as is in my opinion.

Reply:
No problem, but you probably should at least add in the missing period.

Reply:
No problem, but you should probably at least add in the missing period.

Reply:
Oh no, I'm failing so terribly...